credsweeper/__main__.py

```diff
@@ -106,6 +106,12 @@ def get_arguments() -> Namespace:
                         const="output.json",
                         dest="json_filename",
                         metavar="PATH")
+    parser.add_argument("--save-xlsx",
```

Comment:
In the report generation code either json or xml is generated.
In the argument parsing there is no info about the mutual exclusivity of these options, isn't it?

Reply:
I fixed it by 7b231b5.
Please check.